data/remnant/remnant missions.txt

```diff
@@ -1061,138 +1061,689 @@ mission "Remnant: Learn Sign Follow Up"
+				goto ask
+			
+			label sprites
+			apply
+				set "deep: named void sprites"
+			`	"Void sprites," Ivan says slowly, as if letting the name sit on his tongue. "Void sprites... I like it."`
+			
+			label ask
+			`	"Do you think I could get a copy of your research?" you ask.`
+			label "sprites copy"
+			branch again
+				"retrieved information" == 2
+			
+			`	"A copy?" Ivan looks off into space for a moment. "Well, I suppose since you helped me so much, I could make an exception for you. I typically don't share findings with those not on my research team until they're finalized. I've been burned one too many times because of that."`
+			`	Ivan leans over to his desk. "This will be for your eyes only."`
+				goto next
+			
+			label again 
+			`	"Again?" Ivan asks. He leans over to his desk. "I like that you're so interested in my research. I suppose it only makes sense with you helping me, but typically merchant captains aren't this interested in following up on odd jobs they do."`
+			
+			label next
+			apply
+				set "remnant: got deep wormhole and sprite research"
+			`	He presses a button on the desk. "Abigail, could you please make a copy of the encrypted folders 'TAB-RP,' 'WLT-RP', and 'GGA-RP' on the Institute network under my name and bring them to my office in a data chip? Thank you."`
+				goto reencrypt
+			
+			label remnant
+			apply
+				"ivan's suspicion" = ( "retrieved information" / 2 * "deep: knew of stones before scientists" ) + "deep: knew of stones before scientists" + 2 * "Deep: Remnant: Engines: declined" + 2 * "Deep: Remnant: Engines: failed" + "Deep: Remnant Surveillance: declined" + "Deep: Remnant Surveillance: failed"
+			
+			branch technology
+				has "remnant: told that deep has technology"
+			
+			branch surveillance
+				has "remnant: told about surveillance"
+			
+			choice
+				`	"I was wondering what your research on those Remnant ship scans has brought up."`
+				`	"Could I get a copy of your research on those Remnant ship scans?"`
+					goto suspicion
+			branch cubes
+				has "Deep: Remnant Surveillance: done"
+			`	"Ah. Well, there's quite a bit to talk about." Ivan reclines his chair.`
+			`	"The ship scans look like nothing we've ever seen before. The ships seem to have a sort of... organic property to them. Here, let me show you." Ivan pulls up a schematic on his computer of the Remnant ships that you scanned. You're somewhat surprised that he could pull so much information from an outfit scan. "The ships look relatively normal for the most part, but the hull of these ships have an almost organic structure to them." He zooms in on a section of the schematic that highlights the hull structure. "Now when it comes to ships designed by say Betelgeuse or even the Navy, the hull supports are arranged in triangular or square sections. But these ships have a pseudo-hexagonal design to them, completely non-uniform. It almost looks like the cell structure of some plants.`
```

Comment:
```suggestion
			`	"The ship scans look like nothing we've ever seen before. The ships seem to have a sort of... organic property to them. Here, let me show you." Ivan pulls up a schematic on his computer of the Remnant ships that you scanned. You're somewhat surprised that he could pull so much information from an outfit scan. "The ships look relatively normal for the most part, but their hulls have an almost organic structure to them." He zooms in on a section of the schematic that highlights the hull structure. "Now, when it comes to ships designed by, say, Betelgeuse, or even the Navy, the hull supports are arranged in triangular or square sections. But these ships have a pseudo-hexagonal design to them, completely non-uniform. It almost looks like the cell structure of some plants.`
```
Removed the double `the(se) ships`, and added some commas. It looks a little crowded with that many commas, but it's grammatically correct, and I like to think it helps the reader imagine the rhythm someone would actually speak the phrase aloud in.